doc/sphinx/configuring/reference/options.md

```diff
@@ -0,0 +1,11 @@
+(options-reference)=
+
+# Mir configuration options
+
+Mir compostiros can usually be configured (in the order of preference) on the command line, in the environment, or in a configuration file.
+
+Different compositors will expose different options, but the below is a common set.
```

Comment:
Rather that "a common set" which is open to misinterpretation, perhaps say "from the miral-shell example"?